diff -urN wp_tray-0.4.6.orig/wp_tray.desktop.in wp_tray-0.4.6/wp_tray.desktop.in
--- wp_tray-0.4.6.orig/wp_tray.desktop.in	2004-12-02 13:55:26.785341344 +0100
+++ wp_tray-0.4.6/wp_tray.desktop.in	2004-12-02 13:55:55.512974080 +0100
@@ -1,10 +1,11 @@
-
 [Desktop Entry]
-Name=wp_tray Version 0.1
-Comment=wp_tray Version 0.1
+Encoding=UTF-8
+Name=wp_tray
+Comment=GNOME wallpaper selection tool
+Comment[pl]=Narzędzie dla GNOME do wyboru tapety
 Exec=wp_tray
-Icon=@PACKAGE_PIXMAPS_DIR@/ 
 Terminal=false
 MultipleArgs=false
 Type=Application
-
+Categories=GTK;GNOME;Settings;DesktopSettings;
+# vi: encoding=utf-8
